autosklearn/ensemble_builder.py

```diff
@@ -314,7 +319,8 @@ def read_ensemble_preds(self):
                     "seed": _seed,
                     "num_run": _num_run,
                     "budget": _budget,
-                    "deleted": False,
+                    "is_new": True,
+                    "modified": os.path.getmtime(y_ens_fn),
```

Comment:
Could you please check whether that's the same as `mtime_ens`?

Reply:
I think it is.
I totally overlooked this entry. Lets use it instead.